src/main/java/core/basesyntax/operators/Divide.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.operators;
+
+import java.math.BigInteger;
+
+import org.apache.log4j.Logger;
+
+public class Divide extends Operators {
+    private static final Logger logger = Logger.getLogger(Divide.class);
+
+    @Override
+    public String compute(Integer a, Integer b) {
```

Comment:
```suggestion
    public String compute(Integer a, Integer b) {
```

Чому в результаті ти повертаєш String, а не числовий тип даних?

Reply:
Тому що в задуму було, що це дані для для юзера, які він отримує через на консоль, а туди зручніше передавати стрінгу, бо саме нею оперують BigInteger.